lib/broccoli/ember-app.js

```diff
@@ -332,18 +332,34 @@ EmberApp.env = function() {
   Provides a broccoli files concatenation filter that's configured
   properly for this application.
 
+  @deprecated
   @method concatFiles
   @param tree
   @param options
   @return
 */
-EmberApp.prototype.concatFiles = function(tree, options) {
+EmberApp.prototype.concatFiles = function(tree, options, ignoreDeprecation) {
+  this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.', ignoreDeprecation);
```

Comment:
Instead of a Boolean here, let's pass a non-exported symbolish value. I'd hate for folks to just get around the warning by passing true...